__tests__/schema/notification.ts

```diff
@@ -2296,34 +2317,68 @@ describe('setNotificationState', () => {
       user: user.id,
     })
     await assertSuccessfulGraphQLMutation({
-      ...createSetNotificationStateMutation({
-        id: 1,
+      ...createSetNotificationsStateMutation({
+        ids: [1],
         unread: false,
       }),
       client,
+      data: { setNotificationsState: null },
     })
-    await assertSuccessfulGraphQLQuery({
-      query: gql`
-        {
-          notifications {
-            nodes {
-              id
-              unread
-            }
-          }
-        }
-      `,
-      data: {
-        notifications: {
-          nodes: [
-            {
-              id: 1,
-              unread: false,
-            },
-          ],
-        },
+  })
+
+  test('unauthenticated', async () => {
+    const client = createTestClient({
+      service: Service.SerloCloudflareWorker,
+      user: null,
+    })
+    await assertFailingGraphQLMutation(
+      {
+        ...createSetNotificationsStateMutation({
+          ids: [1, 2, 6],
+          unread: false,
+        }),
+        client,
       },
-      client,
+      (errors) => {
+        expect(errors[0].extensions?.code).toEqual('UNAUTHENTICATED')
+      }
+    )
+  })
+
+  test('wrong user id', async () => {
+    global.server.use(
+      rest.post(
+        `http://de.${process.env.SERLO_ORG_HOST}/api/set-notification-state/1`,
+        (req, res, ctx) => {
+          return res(ctx.status(403), ctx.json({}))
+        }
+      )
+    )
+    const client = createTestClient({
+      service: Service.SerloCloudflareWorker,
+      user: user2.id,
     })
+    await assertFailingGraphQLMutation(
+      {
+        ...createSetNotificationsStateMutation({ ids: [1], unread: false }),
+        client,
+      },
+      (errors) => {
+        expect(errors[0].extensions?.code).toEqual('FORBIDDEN')
+      }
+    )
   })
+
+  function createSetNotificationsStateMutation(
+    variables: MutationSetNotificationsStateArgs
+  ) {
+    return {
+      mutation: gql`
+        mutation setNotificationsState($ids: [Int!]!, $unread: Boolean!) {
+          setNotificationsState(ids: $ids, unread: $unread)
+        }
+      `,
+      variables,
+    }
+  }
```

Comment:
Can be a 

```suggestion
const mutation = gql`
        mutation setNotificationsState($ids: [Int!]!, $unread: Boolean!) {
          setNotificationsState(ids: $ids, unread: $unread)
        }
      `
```

at the beginning of the describe block.